### DISCUSSION TITLE
Why sometimes generate command create a 200.html and sometime create index.html?

### DISCUSSION THREAD

**Discussion by renanjoppert1** [nuxt/nuxt#10636] [General]
I don't understand when execute npm run generate in a project, sometime the build create 200.html file and another time create index.html ?

**Comment by AbdullahAlhariri** [comment]
In Nuxt.js, the **generate** command is used to generate a static version of your website. By default, this command will create an **index.html** file for every route in your application. However, if you have configured Nuxt to use a different file name for your index file (for example, if you have set the **generate.dir** option in your nuxt.config.js file to **200.html**), then the **generate** command will create a 200.html file for each route instead of an **index.html** file. The specific file name that is used for the generated index file will depend on the configuration options that you have set in your nuxt.config.js file.
